ee/tabby-webserver/src/schema/auth.rs

```diff
@@ -301,6 +302,39 @@ pub struct PasswordResetInput {
     pub password2: String,
 }
 
+#[derive(Validate, GraphQLInputObject)]
+pub struct PasswordUpdateInput {
+    pub old_password: String,
```

Comment:
```suggestion
    pub old_password: Option<String>,
```

as discussed this need to be optional (since for user.is_password_set, there's no old password)